public/video-ui/src/reducers/configReducer.js

```diff
@@ -0,0 +1,11 @@
+export default function config(state = false, action) {
+  switch (action.type) {
+
+    case 'CONFIG_RECEIVED':
+        return action.config || false;
```

Comment:
Actually, I'm taking that back, the config is set to an empty object when it's added to the store if there isn't one. We could just remove the check